site/docs-md/cordova/migrating-from-cordova-to-capacitor.md

```diff
@@ -0,0 +1,130 @@
+# Migrating from Cordova/PhoneGap to Capacitor
```

Comment:
The guide looks to be very focused on Ionic, but Capacitor is framework agnostic, and people migrating from Cordova/Phonegap might not be using Ionic.

Reply:
rewriting a bit to fix that, good call. need to have a balance

Reply:
you're right! good catch. i reworked it.